services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Implement exception handling to manage scenarios where the `MovieSession` with the given ID is not found. This will prevent potential runtime errors during deletion.